contracts/0.8.25/vaults/predeposit_guarantee/PredepositGuarantee.sol

```diff
@@ -573,11 +526,11 @@ contract PredepositGuarantee is IPredepositGuarantee, CLProofVerifier, PausableU
         balance.total -= PREDEPOSIT_AMOUNT;
         balance.locked -= PREDEPOSIT_AMOUNT;
 
-        (bool success, ) = _recipient.call{value: PREDEPOSIT_AMOUNT}("");
+        (bool success, ) = address(stakingVault).call{value: PREDEPOSIT_AMOUNT}("");
         if (!success) revert CompensateFailed();
 
-        emit BalanceCompensated(nodeOperator, _recipient, balance.total, balance.locked);
-        emit ValidatorCompensated(_validatorPubkey, nodeOperator, address(stakingVault), _recipient);
+        emit BalanceCompensated(nodeOperator, address(stakingVault), balance.total, balance.locked);
+        emit ValidatorCompensated(_validatorPubkey, nodeOperator, address(stakingVault));
```

Comment:
we can add a pubkey param to `BalanceCompensated` and get rid of `ValidatorCompensated`